bibtex-actions.el

```diff
@@ -355,8 +356,7 @@ will then return the key 'key', resulting in '@key' at point.
 Supports the pandoc and 'org-cite' key syntax, in either
 'org-mode' or 'markdown-mode'."
     ; FIX
-    (when (and (or (eq major-mode 'org-mode)
-                   (eq major-mode 'markdown-mode))
+    (when (and (looking-back org-element-citation-prefix-re (line-beginning-position))
```

Comment:
With this line, or the two is replaces, this does not work.

With all removed, and only the below, it does work.

@ilupin since you've been able to contribute org-cite code, am I correct you've managed to get the branch working locally?

Can you see if you can figure this out please? I am thinking the problem is either a) my org-cite install, or b) something fundamental about the elisp here (it doesn't have to do with the capf-specific details).